src/index.js

```diff
@@ -124,7 +124,12 @@ export {
 
 // shader modules
 export {default as fp32} from './shadertools/modules/fp32/fp32';
-export {default as fp64} from './shadertools/modules/fp64/fp64';
+export {
```

Comment:
I was thinking the utils would be accessible through the shader module export, rather than us having to explicitly export every util from every shader module.But maybe this